things/service.go

```diff
@@ -411,10 +405,10 @@ func (svc service) Share(ctx context.Context, token, id, relation string, userid
 	}
 	res, err := svc.auth.AddPolicies(ctx, &policies)
 	if err != nil {
-		return errors.Wrap(errAddPolicies, err)
+		return errors.Wrap(svcerr.ErrUpdateEntity, err)
 	}
 	if !res.Added {
-		return err
+		return errors.Wrap(svcerr.ErrUpdateEntity, err)
```

Comment:
Don't wrap the error since it has already been handled above just return `svcerr.ErrAddPolicies`

Reply:
We do not want ```svcerr.ErrAddPolicies``` to be sent to a user, that is why we send ```svcerr.ErrCreateEntity``` which is what the user will see ```errAddPolicies``` is left as an internal error which will be logged but we do not want it to be sent along with the response